app/js/lib/validation_dispatchers/base.js

```diff
@@ -20,6 +20,7 @@ var _ = require( 'underscore' ),
 		finishActionCreationFunction: null,
 		beginActionCreationFunction: null,
 		fields: null,
+		fieldValuesIfNotSet: null,
```

Comment:
I was a bit confused by this, since it's never written to, until I realized this is a base class. (Such code sharing via inheritance!) I think it'd be nice to have the expected type indicated here, so you don't need to go look at derivative classes to see what's expected.